### PR TITLE
Add fee provider specification

### PR DESCRIPTION
**Pull request by rantan**
After merged this PR, I'm going to impment Fee Provider feature acoding to this specification.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,6 +157,91 @@ bin/rails glueby:contract:timestamp:confirm
...
+
+```ruby
+Glueby.fee_mode = :fee_provider_bears
+Glueby::FeeProvider.configure({
+  # The fee that Fee Provider pays on each transaction.
+  fixed_fee: 1000,
+  # Fee Provider tries to keep the number of utxo in utxo pool as this size using `glueby:fee_provider:manage_utxo_pool` rake task
+  utxo_pool_size: 20
+})
+```
```
Comment (azuchi) [inline README.md:188]: followin are better:

```
Glueby::FeeProvider.configure do |c|
  c.fixed_fee = 1000
  c.utxo_pool_size = 20
end
```

File: README.md [hunk 2]
```diff
@@ -157,6 +157,91 @@ bin/rails glueby:contract:timestamp:confirm
...
+
+2. Deposit TPC into Fee Provider's wallet
+
+Get an address of the wallet.
```
Comment (azuchi) [inline README.md:192]:
```suggestion
Get an address from the wallet.
```

File: README.md [hunk 3]
```diff
@@ -157,6 +157,91 @@ bin/rails glueby:contract:timestamp:confirm
...
+
+Configuration:
+  fixed_fee = 1000
+  utxo_pool_size = 20
+```
+
+If you want to get the status information, you can use the `status` task.
+
+```shell
+$ bundle exec rake glueby:fee_provider:status
```
Comment (azuchi) [inline README.md:242]: There should be output here as well.

**Review by azuchi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rantan) [reply README.md:188]: How do you like to create a new configuration method for whole Glueby modules, If you want to improve the configuration ways?

like this:

```ruby
Glurby.configuration do |c|
  c.wallet_adapter = :active_record
  
  c.rpc_config = { schema: 'http', host: '127.0.0.1', port: 12381, user: 'user', password: 'pass' }
  
  # Use FeeProvider to supply inputs for fees on each transaction that is created on Glueby.
  c.fee_provider_bears!
  c.fee_provider_config = { fixed_fee: 1000, utxo_pool_size: 20 }
end
```

File: README.md (hunk 3 above)
Comment (rantan) [reply README.md:242]: fixed at 1c3105b

File: README.md (hunk 1 above)
Comment (azuchi) [reply README.md:188]: LGTM

Comment (rantan) [reply README.md:188]: Should I do it in this PR or create another PR?

Comment (azuchi) [reply README.md:188]: You can update this PR as spec. Implementation is another one.

Comment (rantan) [reply README.md:195]: update spec at ddfae69

**Review by azuchi — APPROVED** [review]

**Pull request merged**